packages/nimble-tokens/source/styledictionary/nimble-extensions.js

```diff
@@ -1,78 +0,0 @@
-/**
```

Comment:
just inlined this stuff into the config.js as we own it now (adobe xd dsp plugin gone) and don't need to manage in a separate file